shared/src/business/useCases/public/getPublicCaseInteractor.ts

```diff
@@ -11,7 +12,7 @@ import { formatPublicCase } from '../../useCaseHelper/consolidatedCases/formatPu
  * @returns {object} the case data
  */
 export const getPublicCaseInteractor = async (
```

Comment:
Should this be moved to web-api/ if appContext is going to be typed that way?

Reply:
Moved!